server/src/com/cloud/api/ApiResponseGsonHelper.java

```diff
@@ -71,4 +83,19 @@ public boolean shouldSkipField(FieldAttributes f) {
             return false;
         }
     }
+
+    private static class LogExclusionStrategy extends ApiResponseExclusionStrategy implements ExclusionStrategy {
```

Comment:
LoggingExclusionStrategy allready exists. It shouldn't be needed to implement again.